tests/phpunit/data/plugins/wordpress-importer/readme.txt

```diff
@@ -75,7 +75,7 @@ an export file is uploaded to a server with bad permissions and WordPress 3.3 or
 = 0.3 =
 * Use an XML Parser if possible
 * Proper import support for nav menus
-* ... and much more, see [Trac ticket #15197](https://core.trac.wordpress.org/ticket/15197)
+* ... and much more, see https://core.trac.wordpress.org/ticket/15197
```

Comment:
This file (really the entire `tests/phpunit/data/plugins/wordpress-importer/` folder) is special.  It comes from an external source, and we should probably update it again later.  So we shouldn't make any changes here.

See #42 for details.